### ISSUE TITLE
Add Italian and Spanish LowerKnowledgeBase

### ISSUE THREAD

**Issue by kermitt2** [kermitt2/nerd#15]
- [x] create DefaultConfigItWp and DefaultConfigEsWp for the mediawiki parser
- [x] generate resources (in grisp project) and loader (here)
- [x] make available LMDB data
- [x] create models

**Comment by lfoppiano** [comment]
We need to test this

**Comment by tantikristanti** [comment]
We have a new issue regarding the test for Italian language. It gives different result of domains as a disambiguation result between 'localhost' and 'nerd.huma-num.fr/test/'. New issue #48

**Comment by lfoppiano** [comment]
For this task we need to check: 
1. the documentaiton is updated
2. the languages are working for text and pdf

**Comment by tantikristanti** [comment]
The result for the testing regarding the issue for Italian and Spanish language:
1. The documentation has been updated in `http://nerd.readthedocs.io/en/0.0.3/restAPI.html?highlight#supported-languages`

2. The service for text and also for Pdf files for Italian and Spanish language works properly only in `http://nerd.huma-num.fr/test/` but not in localhost
a. Italian language
- huma-num test link
<img width="838" alt="it text" src="https://user-images.githubusercontent.com/28044697/34986524-3416474a-fab8-11e7-9368-b71f13482919.png">

<img width="823" alt="it pdf" src="https://user-images.githubusercontent.com/28044697/34986539-3dd3f57a-fab8-11e7-890a-d09a225a96ec.png">

- localhost
<img width="827" alt="it text local" src="https://user-images.githubusercontent.com/28044697/34986542-4317c66a-fab8-11e7-9b29-a82f6475d2ce.png">
<img width="737" alt="it pdf local" src="https://user-images.githubusercontent.com/28044697/34986545-46d8ddfc-fab8-11e7-81a3-c22ec55342f4.png">


b. Spanish language
- huma-num test link
<img width="841" alt="es text" src="https://user-images.githubusercontent.com/28044697/34986552-4c98813e-fab8-11e7-8eb0-1422d9c55ebc.png">


<img width="725" alt="es pdf" src="https://user-images.githubusercontent.com/28044697/34986574-58094b48-fab8-11e7-8260-5d283877591c.png">

- localhost

<img width="852" alt="es text local" src="https://user-images.githubusercontent.com/28044697/34986593-5fec5e86-fab8-11e7-962c-19ef9871bd26.png">

<img width="723" alt="es pdf local" src="https://user-images.githubusercontent.com/28044697/34986602-659262fe-fab8-11e7-8e20-ff86731ccd3f.png">

**Comment by lfoppiano** [comment]
@tantikristanti I don't understand, what doesn't work? 
I can say that works fine on my machine actually.

**Comment by tantikristanti** [comment]
This morning, I have updated the latest branch 0.0.3 since I saw some changes. And even after lunch I re-build the latest one. 

<img width="565" alt="screen shot 2018-01-16 at 13 32 38" src="https://user-images.githubusercontent.com/28044697/34989227-0273bbe6-fac2-11e7-98fc-c06f03ecbed1.png">

The disambiguation service works only with English for now on in local machine, even it works properly in huma-num test.

This is the log for Spanish language that gave nothing as results.
<img width="1409" alt="screen shot 2018-01-16 at 13 37 09" src="https://user-images.githubusercontent.com/28044697/34989338-735a534c-fac2-11e7-85a1-1cf6b7307cda.png">
